examples/api/test/material/flexible_space_bar/flexible_space_bar.0_test.dart

```diff
@@ -0,0 +1,41 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:io';
+
+import 'package:flutter/material.dart';
+import 'package:flutter_api_samples/material/flexible_space_bar/flexible_space_bar.0.dart' as example;
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  // The app being tested loads images via HTTP which the test
+  // framework defeats by default.
+  setUpAll(() {
+    HttpOverrides.global = null;
+  });
+
+  testWidgets('The app bar should stretch when over-scrolled', (WidgetTester tester) async {
+    await tester.pumpWidget(
+        const example.FlexibleSpaceBarExampleApp(),
+    );
+
+    expect(find.text('Flight Report'), findsOne);
+
+    expect(find.widgetWithText(ListTile, 'Sunday'), findsOne);
+    expect(find.widgetWithText(ListTile, 'Monday'), findsOne);
+    expect(find.text('sunny, h: 80, l: 65'), findsExactly(2));
+    expect(find.byIcon(Icons.wb_sunny), findsExactly(2));
+
+    final Finder appBarContainer = find.byType(Image);
+    final Size sizeBeforeScroll = tester.getSize(appBarContainer);
+    final Offset target = tester.getCenter(find.byType(ListTile).first);
+    final TestGesture gesture = await tester.startGesture(target);
+    await gesture.moveBy(const Offset(0.0, 100.0));
+    await tester.pump(const Duration(milliseconds: 10));
+    await gesture.up();
+    final Size sizeAfterScroll = tester.getSize(appBarContainer);
+
+    expect(sizeBeforeScroll.height, lessThan(sizeAfterScroll.height));
+  });
```

Comment:
Can you try to activate this test on all platforms?
(Maybe with a comment mentioning this is to test that ScrollBehavior.dragDevices is correctly set).

Reply:
Sure! Is [test: Activate all platforms](https://github.com/flutter/flutter/pull/157107/commits/af9653e54d5729f96fa14910d47f39a7ebb96511) what you expected ?